viziphant/patterns.py

```diff
@@ -429,11 +430,22 @@ def plot_patterns_hypergraph(patterns, num_neurons=None):
         :func:`elephant.spade.spade` or
         :func:`elephant.cell_assembly_detection.cell_assembly_detection`
         pattern detectors.
+    node_size (optional): int
+        Change the size of the drawen nodes
+    pattern_size (optional): range
+        Only draw patterns that are in range of pattern_size
     num_neurons: None or int
         If None, only the neurons that are part of a pattern are shown. If an
         integer is passed, it identifies the total number of recorded neurons
         including non-pattern neurons to be additionally shown in the graph.
         Default: None
+    must_involve_neuron (optional) : int
+        Highlight pattern which includes neuron x
+    node_color (optional) : String
+        change the color of the nodes
+
+    node_linewidth (optional) : int
+            change the line width of the nodes
```

Comment:
I think the order of the individual options seems a bit random. I think users will expect first the options that control *what* is shown (e.g., number of neurons), then *how* its shown (e.g., node color).